api/src/test/java/org/pmiops/workbench/testconfig/FakeTicker.java

```diff
@@ -1,17 +1,19 @@
 package org.pmiops.workbench.testconfig;
 
 import com.google.common.base.Ticker;
+import java.util.concurrent.TimeUnit;
 
 public class FakeTicker extends Ticker {
-  private final long elapsedMillis;
+  private long elapsedMillis;
 
-  protected FakeTicker(long elapsedMillis) {
+  protected FakeTicker(long tickNanos) {
     super();
-    this.elapsedMillis = elapsedMillis;
+    this.elapsedMillis = tickNanos;
```

Comment:
no

Reply:
If I had a dollar for every timestamp and duration type I've had to use in this project, I'd have $150 Eastern/$149 Central.